src/standard/utils.html

```diff
@@ -306,6 +306,17 @@
         }
       }
       return elt;
+    },
+
+    /**
+     * Checks whether an element is in this element's shadow DOM tree.
+     *
+     * @method isShadowDomChild
+     * @param {HTMLElement=} node The element to be checked.
+     * @return {Boolean} true if node is in this element's shadow DOM tree.
+     */
+    isShadowDomChild: function(node) {
+      return Polymer.dom(this).getOwnerRoot() !== Polymer.dom(node).getOwnerRoot();
```

Comment:
I'm a little worried that the use case is getting lost in the shuffle here. Can someone clarify what the current direction is, so that I can give a meaningful response?